subfun/BIDS_RFX.m

```diff
@@ -279,44 +286,43 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
             if ~isempty(findstr(' ',Session(j).con))
                 Session(j).con(findstr(' ',Session(j).con)) = '';
             end
-            
-            cd(fullfile(opt.derivativesDir,RFX_FolderName))
+
```

Comment:
The variable RFX_FolderName already includes the full path. 
so "fullfile(opt.derivativesDir,RFX_FolderName)" --> gives an error because the path is repeated in the same string

So I replaced "fullfile(opt.derivativesDir,RFX_FolderName)"  --> RFX_FolderName
This is will be repeated for a couple of times in this script